### PR TITLE
GCI-33 - Revamp site footer

### PR DESCRIPTION
**Pull request by Izaron**
Ticket: http://issues.openmrs.org/browse/GCI-33
Melange task: https://www.google-melange.com/gci/task/view/google/gci2014/5744380010823680

### CODE REVIEW

File: views/layouts/page/bottom.ejs [hunk 1]
```diff
@@ -1,5 +1,5 @@
 <div class="clear"></div>
 <%- bodyAppend %>
-
+	<div id="about"><%- aboutHTML %></div>
```
Comment (elliottwilliams) [inline views/layouts/page/bottom.ejs]: Can you move this above `bodyAppend`? `bodyAppend` can be used to load scripts that run synchronously, which will cause the browser to noticeably pause at those scripts while they load/run. This means the page may not render `aboutHTML` for a moment, which in general looks weird :)

**Comment by Izaron** [comment]
Ok, I done it

**Comment by elliottwilliams** [comment]
Thanks!

> This task has been approved, but will not be merged into the repository until all GCI tickets in the ID Dashboard UI Revamp project have been submitted. We're doing this so that multiple students can submit their design implementation and we can select from the best collective implementation.

**Comment by robbyoconnor** [comment]
@Izaron  could you resolve merge conflicts and then rebase and squash them into one commit.

**Comment by Izaron** [comment]
@robbyoconnor of course :)
https://github.com/openmrs/openmrs-contrib-id/pull/69

**Comment by robbyoconnor** [comment]
Why is this closed?

**Pull request closed without merging**